frontend/src/components/Analysis/utils.tsx

```diff
@@ -58,10 +60,32 @@ export function formatBucketName(unformattedName: Array<number>): string {
   }
 }
 
+export function parseComboCountFeatures(
+  comboCounts: ComboCount[],
+  levelName: string,
+  analysisName: string,
+  featureDescription: string
+) {
+  const parsedResult: { [metricName: string]: ResultFineGrainedParsed } = {};
+  // TODO (Chonghan) should not hardcode here.
```

Comment:
`ComboCount` analysis does not have a metrics field. Is it because it should always belong to `Accuracy`?

Reply:
Hi, @PaulCCCCCCH  `Confusion Matrix` should be only defined w.r.t classification problem.

Reply:
ComboCount could conceivably be used for things other than confusion matrices. For example, if we set `feature = ["text_topic", "predicted_label"]` it could calculate the predictive distribution for each topic in a text.

I'd just suggest displaying `ComboCount` regardless of the metric.

Reply:
Now you can see the figures for `ComboCount` no matter which metric tab you are in.